challenge_templates/pwn/Dockerfile

```diff
@@ -22,7 +22,7 @@ ADD challenge/flag /flag
 ADD challenge/challenge /home/ctf/challenge
 
 # Set some proper permissions
-RUN chown -R root:ctf /home/ctf
+RUN chown -R root:ctf /home/ctf /flag
 RUN chmod 750 /home/ctf/challenge
 RUN chmod 750 /home/ctf/run_challenge.sh
 RUN chmod 440 /flag
```

Comment:
Is this line conflicting with your change on line 25?

Reply:
In the docker compose? Nope, this is just the standard linux permissions, the other mounts the root file system with `ro` but all of these still apply

Reply:
Oh wow, I taught you were doing `chmod -r`, not `chown`...